src/firejail/profile.c

```diff
@@ -113,12 +113,65 @@ void profile_add_ignore(const char *str) {
 }
 
 
+int profile_check_conditional(char *ptr, int lineno, const char *fname) {
+	struct cond_t {
+		char *name;	// conditional name
+		size_t len;	// length of name
```

Comment:
having to manually specify the length seems a bit fragile, when you anyway have only constant strings.
i would suggest to drop the len from the struct and get the length with strlen when needed.

Reply:
If you look at `profile_check_line`, every 5th line has the problem of using literal numerals instead of strlen.  I don't think it would be an exaggeration to say that its pervasive.  However, don't like the error prone aspect either.  I've updated the pull request branch to get rid of the literal, but not the len field.